sklearn/linear_model/base.py

```diff
@@ -363,16 +376,29 @@ def fit(self, X, y):
         y : numpy array of shape [n_samples, n_targets]
             Target values
 
+        sample_weight : float or numpy array of shape [n_samples]
```

Comment:
Same comment here about a scalar `sample_weight`. Sorry if I'm missing something here, but intuitively I would expect that the result would not change if you weighted all samples by the same amount.